templates/hotgraphicPopup.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="hotgraphic-popup-inner clearfix">
     {{#each _items}}
-    <div class="hotgraphic-item component-item clearfix item-{{@index}} {{_classes}}">
+    <div class="hotgraphic-item component-item clearfix {{_classes}}" data-index={{@index}}>
```

Comment:
Think we need `active` and `visited` here classes to cater for already visited items? Might make the `onOpened()` logic in the popup view a bit easier.